src/cmd/singularity/cli/push.go

```diff
@@ -32,15 +31,11 @@ var PushCmd = &cobra.Command{
 	Args:   cobra.ExactArgs(2),
 	PreRun: sylabsToken,
 	Run: func(cmd *cobra.Command, args []string) {
-		if authWarning != auth.WarningEmptyToken &&
-			authWarning != auth.WarningTokenToolong &&
-			authWarning != auth.WarningTokenTooShort {
-			if authToken != "" {
-				libexec.PushImage(args[0], args[1], PushLibraryURI, authToken)
-				return
-			}
+		if authWarning == "" || authToken != "" {
```

Comment:
Same here, should it ever proceed if `authWarning` is non-empty? For example, if `authWarning` is `auth.WarningTokenTooShort`? The logic here used to `Fatalf` in this case, was this change intentional?